ros/resources/v0/hosts.py

```diff
@@ -53,6 +53,9 @@ def get(self):
             else:
                 host = list(filter(lambda host: host['id'] == str(i.inventory_id), inv_hosts['results']))[0]
                 host['performance_score'] = i.__dict__['performance_score']
+                host['performance_score']['memory_score'] //= 20
```

Comment:
Will it be possible to move or add properties at model level?

Reply:
@semtexzv any thoughts here?

Reply:
With `@property` also it is possible to define custom field at model level.
example -

    @property
    def score(self):
        print("Getting value...")
        return 3

the issue was due to `marshal_with`. In this fields, forgot to declare field name and type for new property.
https://github.com/RedHatInsights/ros-backend/blob/main/ros/resources/v0/hosts.py#L27after adding new property details under hosts_fields, it is giving property details in response as well.

@apuntamb, could you please make changes and test it locally if that works for you.

Reply:
+1 for @kgaikwad suggestion of having `@property ` inside model.